.github/actions/maven-build-and-tag/action.yml

```diff
@@ -157,6 +168,11 @@ runs:
         MAVEN_USERNAME: ${{ inputs.maven-username }}
         MAVEN_PASSWORD: ${{ inputs.maven-password }}
 
+    - name: Remove running docker containers
+      if: inputs.reuse-testcontainers == 'true'
+      shell: bash
+      run: docker rm -f $(docker ps -a -q)
```

Comment:
Quick note: I'm wondering about potential termination of containers that would not have been started by testcontainers, but maybe we don't have such use cases.
It seems that there's no easy way to isolate them anyway (unless we added specific labels in tests for each container...) according to https://github.com/testcontainers/testcontainers-java/issues/3569

Reply:
Thank you @atchertchian! Yes, I was also wondering about that, but AFAIK we don't have yet such use-cases. It would be necessary if we had though - something to consider in the future.